wp-content/themes/redmond/web/scss/modules/_call-to-action.scss

```diff
@@ -1,23 +1,36 @@
 // ready to schedule row in home page
 .call-to-action {
   background-color: $secondary-color;
-  padding: rem-calc(60px) 0;
+  padding: rem-calc(60px) rem-calc(40px);
+  text-align: center;
+  @include breakpoint(large) {
+    text-align: left;
+  }
   .row {
     position: relative;
+    vertical-align: middle;
   }
   .heading {
+    padding-bottom: rem-calc(20px);
```

Comment:
remove the padding-bottom on large. let the padding on `call-to-action` do the spacing on large.